pyro/contrib/gp/models/gpr.py

```diff
@@ -11,20 +11,33 @@
 
 
 class GPRegression(GPModel):
-    """
-    Gaussian Process Regression module.
+    r"""
+    Gaussian Process Regression model.
```

Comment:
Add a self-contained exposition of GP regression.  Maybe copy the one from `GPModel` and put it here?